libs/Helpers.py

```diff
@@ -7,4 +8,10 @@ class Helpers:
     def text_to_list(self, text):
         rows = text.split('\n')
         print(rows)
-        return rows
\ No newline at end of file
+        return rows
+
+    @keyword
+    def gte(self, version, target):
+        version=tuple(version.translate(str.maketrans('', '', string.punctuation)))
```

Comment:
Does it work with versions like "1.2.0-2"? If the "-2" part is a problem maybe you could use the semver library to parse it and to make the comparisons
https://python-semver.readthedocs.io/en/latest/usage.html

Reply:
From my testing, it does. (1,2,0,2) evaluates as smaller than (1,3,0). To the best of my knowledge tuple comparisons are implemented positions by position

Reply:
Implemented semver in new commit